example/test_problem/ELBDM/HaloMerger/Make_ParCloud_ExtPotTable.py

```diff
@@ -0,0 +1,94 @@
+
+
+###################################################################################################################
+def Potential_UniDenSph(r, M, R):
+   return -NEWTON_G*M/r if r >= R else NEWTON_G*M/(2.0*R*R*R)*(r*r-3.0*R*R)
+###################################################################################################################
+
+
+###################################################################################################################
+# parameters for the external potential
+# the default is the potential of an uniform density sphere with 3x mass and 0.3x radius of the default CDM halo
+ParCloud_ExtPot_UniDenSph_M = 3.0*3.618847000e-02  # in code_mass
+ParCloud_ExtPot_UniDenSph_R = 0.3*3.234605475e-02  # in code_length
+###################################################################################################################
+
+
+###################################################################################################################
+# output the information
+print( 'Information'                                                                            )
+print( 'ParCloud_DensProf_Filename  =   '+ParCloud_DensProf_Filename                            )
+print( 'UNIT_L                      = {: >16.8e} cm'.format(      UNIT_L                      ) )
+print( 'UNIT_D                      = {: >16.8e} g/cm**3'.format( UNIT_D                      ) )
+print( 'UNIT_T                      = {: >16.8e} s'.format(       UNIT_T                      ) )
+print( 'ParCloud_ExtPot_UniDenSph_M = {: >16.8e} UNIT_M'.format(  ParCloud_ExtPot_UniDenSph_M ) )
+print( 'ParCloud_ExtPot_UniDenSph_R = {: >16.8e} UNIT_L'.format(  ParCloud_ExtPot_UniDenSph_R ) )
+###################################################################################################################
+
+
+###################################################################################################################
+# create the external potential table
+ParCloud_DensProf_radius, _     = np.loadtxt( ParCloud_DensProf_Filename, skiprows=1, unpack=True )
+ParCloud_ExtPot_table_radius    = np.append( ParCloud_DensProf_radius, 2.0*ParCloud_DensProf_radius[-1] ) # probably a bug, we need to add one extra row, which will not be used
+ParCloud_ExtPot_table_potential = np.array([Potential_UniDenSph( radius, ParCloud_ExtPot_UniDenSph_M, ParCloud_ExtPot_UniDenSph_R ) for radius in ParCloud_ExtPot_table_radius ])
+###################################################################################################################
+
+
+###################################################################################################################
+# save to file
+np.savetxt( 'ParCloud_ExtPotTable',
+            np.column_stack( (ParCloud_ExtPot_table_radius, ParCloud_ExtPot_table_potential) ),
+            fmt='          %9.8e',
+            header='                     r                 potential' )
```

Comment:
How about using this format?
```suggestion
np.savetxt( 'ParCloud_ExtPotTable',
            np.column_stack( (ParCloud_ExtPot_table_radius, ParCloud_ExtPot_table_potential) ),
            fmt='%23.8e',
            header='%21s %23s'%('r', 'potential') )
```
This can make it easier to maintain the text file.

Reply:
Done. Thanks!